cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/DynamicModel.java

```diff
@@ -11,6 +11,10 @@ public DynamicModel() {
         parameters = new HashMap<>();
     }
 
+    public Map<String, Object> getParameters() {
+        return parameters;
```

Comment:
Couldn't we avoid this? There is a type enforcing getter like: public <T> T getParameter(String key, Class<T> clazz)